news.xql

```diff
@@ -33,17 +33,23 @@ return
             <li>
                 <div class="date">
                 {
-                    if ($age < xdt:dayTimeDuration("PT1H")) then
-                        minutes-from-duration($age) || " minutes ago"
-                    else if ($age < xdt:dayTimeDuration("P1D")) then
-                        hours-from-duration($age) || " hours ago"
-                    else if ($age < xdt:dayTimeDuration("P14D")) then
-                        days-from-duration($age) || " days ago"
+                if ($age < xdt:dayTimeDuration("PT1H")) then
```

Comment:
As long as you're touching this code, please change `xdt` to `xs` - the former is an ancient form of this namespace prefix and is only included in the eXist codebase for "backward compatibility" - https://github.com/eXist-db/exist/blob/develop/exist-core/src/main/java/org/exist/xquery/XQueryContext.java#L3093. Using it in other XQuery processors - like BaseX and Saxon - raises a "namespace xdt has not been declared" error.

Reply:
Perhaps this comment / plan should go in a new pull request.